database/Reviews.js

```diff
@@ -8,6 +8,7 @@ const db = require('./index.js');
 mongoose.Promise = global.Promise;
 
 const ReviewSchema = new Schema({
+  _id: Number,
```

Comment:
If we are going to replace the built in primary key _id, then we really should make sure this number is unique. 

This is because every mongodb document needs a unique key in order to exist in the database. 

Maybe it's better we don't mess with mongo's automatic _id field?

One option is we can leave _id alone and make another key and call it like, "itemID" ?
Then if we have a collision, it won't cause a big mess? 

If we go this route, we may need to write functions that handle the automatic numbering and incrementing of the _id value to ensure database is numbered properly.

Reply:
this is not updated version it is now changed to item_id like i showed you earlier in the morning but i will new pull request on it